src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -1431,40 +1676,101 @@
         }
       }
     },
-    "Comment": {
+    "CashAdvance": {
       "type": "object",
       "properties": {
-        "author": {
-          "description": "Author of the comment",
-          "$ref": "#/definitions/Employee"
+        "amountRequested": {
+          "description": "The amount of Cash Advance, expressed in the currency of this Cash Advance",
+          "$ref": "#/definitions/Amount"
         },
-        "creationDateTime": {
+        "approvalStatus": {
+          "description": "The approval status of the Cash Advance",
+          "$ref": "#/definitions/CashAdvanceApprovalStatus"
+        },
+        "cashAdvanceId": {
           "type": "string",
-          "description": "Creation date of the comment (in the format YYYY-MM-DD HH:mm:ss.SSS)"
+          "description": "The unique identifier of the Cash Advance"
         },
-        "isLatest": {
-          "type": "boolean",
-          "example": false,
-          "description": "Is set to true if the comment was edited after the last workflow transition"
+        "comment": {
+          "type": "string",
+          "description": "Last comment of this Cash Advance"
         },
-        "value": {
+        "exchangeRate": {
```

Comment:
Please use the same definition text as the markdown files, especially for this field.

Reply:
Done